### ISSUE TITLE
Starting embedded node with `http.enabled: false` crashes with NPE

### ISSUE THREAD

**Issue by nvartolomei** [sksamuel/elastic4s#781]
In particular because of this:

> https://github.com/sksamuel/elastic4s/blob/master/elastic4s-embedded/src/main/scala/com/sksamuel/elastic4s/embedded/LocalNode.scala#L36

```scala
val ipAndPort: String = client().admin().cluster().prepareNodesInfo(nodeId).get()
    .getNodes.iterator().next().getHttp.address().publishAddress().toString
```

**Comment by sksamuel** [comment]
What settings are you using? Because it works if you use the defaults.

**Comment by nvartolomei** [comment]
@sksamuel https://github.com/sksamuel/elastic4s#embedded-node

Not a big trouble though

**Comment by sksamuel** [comment]
The embedded node standard options do work as its used in all the unit tests. You're not doing anything extra?

**Comment by sksamuel** [comment]
Fixed in 5.2.12
